### PR TITLE
Add message sound for received unread messages

### PR DESCRIPTION
**Pull request by Hepic**

### CODE REVIEW

**Comment by dionyziz** [comment]
Do we have copyright information on this mp3 file? If it's directly from Facebook I don't think we're allowed to use it.

File: client/message/history.jsx [hunk 1]
```diff
@@ -87,6 +87,9 @@ const History = React.createClass({
                 this.setState({
                     unread: this.state.unread + 1
                 });
+
+                var audio = new Audio('message_sound.mp3');
+                audio.play();
```
Comment (dionyziz) [inline client/message/history.jsx]: Why reload the mp3 file every time you want to play it?

File: client/message/history.jsx [hunk 2]
```diff
@@ -87,6 +87,9 @@ const History = React.createClass({
                 this.setState({
                     unread: this.state.unread + 1
                 });
+
+                var audio = new Audio('message_sound.mp3');
```
Comment (dionyziz) [inline client/message/history.jsx]: Why reload the mp3 file every time you want to play it?

File: client/message/history.jsx [hunk 3]
```diff
@@ -28,7 +28,8 @@ const History = React.createClass({
             messages: {},
             unread: 0,
             active: true,
-            myUsername: null
+            myUsername: null,
+            audio: new Audio('static/sounds/message_sound.mp3')
```
Comment (gtklocker) [inline client/message/history.jsx]: Does this ever change? If not it should be in the prop instead of the state.

File: client/ting.jsx [hunk 4]
```diff
@@ -127,6 +127,8 @@ const Ting = React.createClass({
         this._socket.emit('login', intendedUsername);
     },
     render() {
+        var audio = new Audio('static/sounds/message_sound.mp3');
+
```
Comment (dionyziz) [inline client/ting.jsx]: This recreates a new audio every time the render function is called. Probably not what you want. React is free to call this function as much as it likes.

**Comment by dionyziz** [comment]
@Hepic Please keep the copyright information for the sound file so that we can put it in our credits later.

**Pull request merged**